src/content/development/release-process/_index.en.md

```diff
@@ -227,6 +227,7 @@ is the source for sharing Kubernetes Operators with the broader community. This
 These are shared with the Kubernetes community via [OperatorHub.io](https://operatorhub.io/).
 * Operators for deployment to OpenShift (community-operators)
 
+We are going to support the Kubernetes community only.
```

Comment:
I would remove this sentence or rephrase it a bit. It's not accurate to say that we are going to support the Kubernetes community only, it's just that as part of the upstream release process we only need to take care of it (and not the OpenShift marketplace one - which is handled separately).

Reply:
Yes, that makes sense.
I still think we should inform the reader that the community-operator is not needed.
If I use `update` instead of `support` does it sound better?
`We are going to update the Kubernetes community operator hub only.`

Reply:
I would phrase it in a non-limiting fashion:

```suggestion
This is only required to support the Kubernetes community. OpenShift users will find the Submariner operator in the official Red Hat catalog.
```

Reply:
Done